app/components/general/components/bc-error-page/macro.njk

```diff
@@ -14,5 +14,10 @@
       <h2 class="nhsuk-heading-2 nhsuk-u-margin-top-5" data-test-id="stackTrace-heading">Stack Trace</h2>
       <p data-test-id="error-stackTrace">{{ params.error.stackTrace | safe }}</p>
     {% endif %}
+    {% if params.error.data %}
+      <pre>
+{{ params.error.data }}
+      </pre>
```

Comment:
This line cannot be indented because the indentation will render.

Reply:
Would formatting it as a single line work?

```nunjucks
<pre>{{ params.error.data }}</pre>
```

Reply:
Good point, I will try it.

Reply:
That does work, sorry for being dumb.

Reply:
No worries 😃